model_zoo/spleen_ct_segmentation_pytorch/README.md

```diff
@@ -0,0 +1,61 @@
+# Description
+Load pre-trained model from MONAI model-zoo with pytorch and train it on a new dataset.
+
+# Overview
+This tutorial shows how to load a pretrained U-net from MONAI model-zoo and train it on BTCV dataset using pytorch. The pretrained U-net was trained on volumetric (3D) segmentation of CT spleen from MSD dataset.
+
+## Data
+The description of data and data preparation can be found in [spleen_ct_segmentation_monai_bundle](../spleen_ct_segmentation_monai_bundle)
```

Comment:
Please note to update the link accordingly when updating the folder and file names.

Thanks.

Reply:
Sure. Thanks!